### PR TITLE
Add support for cosmos messages 

### PR DESCRIPTION
**Pull request by BjrInt**
~~:warning: Not working as is!~~

~~Signature from the [Cosmjs Typescript SDK](https://github.com/cosmos/cosmjs) differs from the previous JS versions. As of now messages are not validated by pyaleph!~~

### CODE REVIEW

**Comment by BjrInt** [comment]
Should be good now! :relieved:

File: src/accounts/cosmos.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        const signDoc = makeSignDoc([aminoMsg], { amount: [], gas: "0" }, "signed-message-v1", "", "0", "0");
+        const { signature } = await this.wallet.signAmino(this.address, signDoc);
+
+        return JSON.stringify(signature);
+    }
+}
+
+async function getCosmosAccount(wallet: OfflineAminoSigner): Promise<CosmosAccount> {
+    const [account] = await wallet.getAccounts();
```
Comment (MHHukiewitz) [inline src/accounts/cosmos.ts:43]: is it possible to select which account to get? Right now, we are getting the first one, no?

Comment (MHHukiewitz) [reply src/accounts/cosmos.ts:43]: Or is the first one also the one the user currently selected through their wallet?

**Review by MHHukiewitz — APPROVED** [review]
LGTM

**Pull request merged**